src/js/tools/metroconfig.ts

```diff
@@ -18,6 +18,11 @@ export interface SentryMetroConfigOptions {
    * @default false
    */
   annotateReactComponents?: boolean;
+  /**
+   * Adds the Sentry replay package for web.
+   * @default false (Mobile) true (Web)
```

Comment:
Since the resolver got more complex due to older Metro versions, let's change this to opt in on all platforms, to avoid breaking builds on accident.

We can update the default later, when we get feedback on the feature.

```suggestion
   * @default true
```